Mathlib/AlgebraicGeometry/Morphisms/ClosedImmersion.lean

```diff
@@ -91,7 +91,7 @@ theorem spec_of_surjective {R S : CommRingCat} (f : R ⟶ S) (h : Function.Surje
 /-- For any ideal `I` in a commutative ring `R`, the quotient map `specObj R ⟶ specObj (R ⧸ I)`
 is a closed immersion. -/
 instance spec_of_quotient_mk {R : CommRingCat.{u}} (I : Ideal R) :
-    IsClosedImmersion (Scheme.specMap (CommRingCat.ofHom (Ideal.Quotient.mk I))) :=
+    IsClosedImmersion (Scheme.Spec.map (CommRingCat.ofHom (Ideal.Quotient.mk I)).op) :=
```

Comment:
What's going on here?

Reply:
Just that `specMap` is now in the AlgebraicGeometry namespace so I chose a random fix.